yorkie/src/main/kotlin/dev/yorkie/document/crdt/CrdtElement.kt

```diff
@@ -10,6 +10,12 @@ internal abstract class CrdtElement(
     movedAt: TimeTicket? = null,
     removedAt: TimeTicket? = null,
 ) {
+    abstract fun toJson(): String
```

Comment:
I think it is better to delay implementing `toJson`, `toSortedJson`, `deepCopy` until they are actually needed.
I'm hoping we could implement these functions easily with JSON serializer/deserializer libraries such as GSON.

Reply:
@skhugh Interesting. How can we use GSON?

Reply:
@hackerwins Please check this page : https://github.com/google/gson
Just add gson dependency to gradle, and use it according to guide docs.
It's the most popular JSON processing library for Java/Kotlin users.

Reply:
Thanks for the introduction. I'd also like to know how to use GSON when creating JSON from Yorkie Document. 🤔